modules/mq-broker/providers.tf

```diff
@@ -1,8 +1,14 @@
 provider "aws" {
```

Comment:
reset this to match https://github.com/cloudposse/terraform-aws-components/blob/main/mixins/providers.depth-1.tf

Reply:
Done